src/steamship/experimental/transports/transport.py

```diff
@@ -63,7 +63,8 @@ def instance_deinit(self, *args, **kwargs):
         end = time.time()
         logging.info(f"Transport deinitialized in {end - start} seconds: {self.__class__.__name__}")
 
-    def send(self, blocks: List[Block], metadata: Dict[str, Any]):
+    def send(self, blocks: List[Block], metadata: Metadata):
+        metadata = metadata or {}
```

Comment:
Maybe I haven't had enough coffee yet, but I don't understand how this makes it optional.  Metadata is still defined as Dict[str, Any], right, not Optional[Dict[str, Any]]?